Singularity/Singularity/Screen/Button.cs

```diff
@@ -112,7 +116,7 @@ public Button(float scale, Texture2D buttonTexture, Rectangle sourceRectangle, V
         /// <param name="buttonText">text that button will appear as</param>
         /// <param name="font"></param>
         /// <param name="position"></param>
-        public Button(string buttonText, SpriteFont font, Vector2 position, bool withBorder = false)
+        public Button(string buttonText, SpriteFont font, Vector2 position, bool withBorder = false, EventArgs eventArgs = default(EventArgs))
```

Comment:
quick question: what exactly is the event args for?

Reply:
that is something I did, its, well, to actually be able to use the event args, since the idea is to create your own event args and pass it, normally one should create a new button class that explicitly passes the given event args class. This way it has to be casted.

Reply:
the default EventArgs is just empty and cannot be used for anything, and since i made it default it doesn't change the behavior of the button if nothing is given